Infra/Indexers/init.sql

```diff
@@ -0,0 +1,13 @@
+create table transfer_usdc(
+    network text,
+    block_hash text,
+    block_number bigint,
+    block_timestamp text,
```

Comment:
```suggestion
  block_timestamp: timestamp,
```

This way we can sort and aggregate by time.